doc/Programming-guide/Key-platform-features/Configuration-management/index.md

```diff
@@ -674,75 +1091,85 @@ The calculated delta update will be an array with three entries, as follows:
+                "testField1":{
+                    "org.kaaproject.configuration.unchangedT":"unchanged"
                 },
-                "testField2": {
-                    "org.kaa.config.testRecordT": {
-                        "testField3": {
-                            "array": [
+                "testField2":{
+                    "org.kaa.config.testRecordT":{
+                        "testField3":{
+                            "array":[
                                 {
-                                    "org.kaa.config.testRecordItemT": {
-                                        "testField4": {
-                                            "int": 4
+                                    "org.kaa.config.testRecordItemT":{
+                                        "testField4":{
+                                            "int":4
                                         },
-                                        "__uuid": "\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0004"
+                                        "__uuid":"\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0004"
                                     }
                                 }
                             ]
                         }
                     }
                 },
-                "testField5": null,
-                "__uuid": "\u0001\u0002\u0003\u0004\u0005\u0006\u0007\b\t\n\u000b\f\r\u000e\u000f\u0010"
+                "testField5":null,
+                "__uuid":"\u0001\u0002\u0003\u0004\u0005\u0006\u0007\b\t\n\u000b\f\r\u000e\u000f\u0010"
             }
         }
     }
 ]
 ```
 
-The first delta item is a partial update for the object with UUID [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3], where the **testField4** field value was changed from 3 to 36. The second delta item is an update for the **testField3** field to remove the item with UUID [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1] (**NOTE:** the **testField1** and **testField5** fields were set to "unchanged"). The last delta item sets the **testField5** field to null and adds a new array item with UUID [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4].
+The first delta item is a partial update for the object with UUID [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3], where the ```testField4``` field value
+was changed from 3 to 36. The second delta item is an update for the ```testField3``` field to remove the item with UUID
+[ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1].
+
+>**NOTE:**
+> The ```testField1``` and ```testField5``` fields were set to "unchanged".
+
+The last delta item sets the ```testField5``` field to ```null``` and adds a new array item with UUID [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4].
 
-## Schema versioning
+# Schema versioning
 
 In the current Kaa version, any configuration schema updates require updates of the client application.
```

Comment:
Add "Using Configuration SDK APIs" section similar to profiles and other programming-guide articles.